deploy/stacks/ses_stack.py

```diff
@@ -0,0 +1,92 @@
+from aws_cdk import (
+    aws_ssm as ssm,
+    aws_kms as kms,
+    aws_sns as sns,
+    aws_ses as ses,
+    aws_route53 as route53,
+    aws_iam as iam,
+    RemovalPolicy,
+)
+
+
+from .pyNestedStack import pyNestedClass
+
+
+class SesStack(pyNestedClass):
+    def __init__(
+            self,
+            scope,
+            id,
+            envname='dev',
+            resource_prefix='dataall',
+            custom_domain=None,
+            **kwargs,
+    ):
+        super().__init__(scope, id,  **kwargs)
+
+        self.KMS_SNS = kms.Key(
+            self,
+            f'{resource_prefix}-{envname}-SNS-key',
+            removal_policy=RemovalPolicy.DESTROY,
+            alias=f'{resource_prefix}-{envname}-SNS-key',
+            enable_key_rotation=True
+        )
+
+        self.sns = sns.Topic(
+            self,
+            f'{resource_prefix}-{envname}-SNS-Email-Bounce-Topic',
+            display_name="SNS-Email-Bounce-Topic",
+            topic_name=f'{resource_prefix}-{envname}-SNS-Email-Bounce-Topic',
+            master_key=self.KMS_SNS,
+        )
+
+        self.sns.apply_removal_policy(RemovalPolicy.DESTROY)
+
+        hosted_zone = route53.HostedZone.from_hosted_zone_attributes(
```

Comment:
If we use email_id instead of custom_domain, and for some reason the custom domain is not there then this piece of code will make the stack fail. We need to add a conditional block and in case it is not there use the email_id in identity below

Reply:
I fixed it in this commit: https://github.com/awslabs/aws-dataall/pull/822/commits/ee2cc20d3ea57a3af053a6d024d413e9826e6102 let me know if you agree with the changes

Reply:
Currently, only the custom_domain is used to verify with SES. My intent was not to verify the email address as identity as, that would mean that the user has to verify the email address and also make couple of entires in their DNS server ( for DKIM and SPF ). 

For example, if the user wants to use an email ids like, data-all-notifications@some-non-aws-domain.com. Then in this case the user has to verify that the email id belongs to him by clicking on the verification email and also authenticate the email address with DKIM and SPF, for which the user needs to have access to the DNS server in someway